src/test/java/com/google/sps/GenerateHuntTest.java

```diff
@@ -367,4 +390,42 @@ public void noDests() {
     int expected = 0;
     Assert.assertEquals(expected, actual);
   }
+
+  @Test
+  /* Get the right cities in datatstore. */
+  public void getRightCities() {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+    Destination disneyWorld =
+        new Destination.Builder()
+            .withName("Disney World")
+            .withCity("Florida")
+            .withObscurity(Destination.Obscurity.EASY)
+            .build();
+
+    String goldenGateJson = new Gson().toJson(goldenGate);
+    String louvreJson = new Gson().toJson(louvre);
+    String disneyJson = new Gson().toJson(disneyWorld);
+
+    Entity goldenGateEnt = new Entity(Constants.DESTINATION_ENTITY);
+    goldenGateEnt.setProperty(Constants.DESTINATION_JSON, goldenGateJson);
+    datastore.put(goldenGateEnt);
+
+    Entity louvreEnt = new Entity(Constants.DESTINATION_ENTITY);
+    louvreEnt.setProperty(Constants.DESTINATION_JSON, louvreJson);
+    datastore.put(louvreEnt);
+
+    Entity disneyEnt = new Entity(Constants.DESTINATION_ENTITY);
+    disneyEnt.setProperty(Constants.DESTINATION_JSON, disneyJson);
+    datastore.put(disneyEnt);
+
+    GetCitiesServlet cities = new GetCitiesServlet();
+    Set<String> actual = cities.getCities();
+
+    Set<String> expected = new HashSet<String>();
+    expected.add("San Francisco");
+    expected.add("Paris");
+    expected.add("Florida");
```

Comment:
you might also want to have a destiantion with the same city to ensure it doesn't return dupes